### ISSUE TITLE
MRAN date based on build day

### ISSUE THREAD

**Issue by ck37** [ucberkeley/bce#57]
Should the MRAN repo be based on the current day that the build is being created, rather than being hardcoded? Or find the most recent MRAN archive to use? http://mran.revolutionanalytics.com/

It seems kind of unfortunate to have to update the URL whenever building or otherwise having packages potentially worth upgrading immediately after the build is created. E.g. right now the MRAN is 2.5 months old.

**Comment by ryanlovett** [comment]
It should be set to a particular day for reproducability. The day of the
build would work.

Ryan

**Comment by ck37** [comment]
Right, for the "most recent MRAN" option it would still be hardcoded in the actual AMI, but it would handle the contingency that there may not be an MRAN folder for the day of the build for whatever random reason.
